packages/shared/src/components/Feed.spec.tsx

```diff
@@ -1058,3 +1073,126 @@ it('should report irrelevant tags', async () => {
     ).not.toBeInTheDocument(),
   );
 });
+
+describe('acquisition form card', () => {
+  const replaceRouter = jest.fn();
+  const url = new URL(`http://localhost:5002?${acquisitionKey}=true`);
+
+  let mockUserAcquisitionVersion = UserAcquisition.V1;
+
+  beforeEach(() => {
+    /* eslint-disable @typescript-eslint/no-var-requires,global-require */
+    const useRouter = jest.spyOn(require('next/router'), 'useRouter');
+
+    const useFeatureOrig = jest.requireActual(
+      './GrowthBookProvider',
+    ).useFeature;
+    jest
+      .spyOn(require('./GrowthBookProvider'), 'useFeature')
+      .mockImplementation((feat) => {
+        if (feat === feature.userAcquisition) {
+          return mockUserAcquisitionVersion;
+        }
+        return useFeatureOrig(feat);
+      });
+    /* eslint-enable @typescript-eslint/no-var-requires,global-require */
+
+    useRouter.mockImplementation(() => ({
+      route: '/',
+      pathname: '',
+      query: { ua: 'true' },
+      asPath: '',
+      push: jest.fn(),
+      events: {
+        on: jest.fn(),
+        off: jest.fn(),
+      },
+      replace: replaceRouter,
+      beforePopState: jest.fn(() => null),
+      prefetch: jest.fn(() => null),
+    }));
+
+    Object.defineProperty(window, 'location', {
+      value: url,
+      configurable: true,
+    });
+  });
+
+  it('should show the form card when the right url query param is present', async () => {
+    renderComponent([createFeedMock()], defaultUser);
+    await screen.findByTestId('acquisitionFormCard');
+  });
+
+  it('should hide the card once form is submitted', async () => {
```

Comment:
Since the test is saying hide the card, we're not actually checking anymore if the card is shown in the end. Maybe we can either say remove query param, or add the test in the end for the card not being found.

Reply:
expect(card).not.toBeInTheDocument();